client/my-sites/subscriptions/index.tsx

```diff
@@ -17,8 +17,10 @@ const SubscriptionManagementPage = () => {
 	const locale = useLocale();
 
 	const SettingsView = () => {
-		const { data: settings } = Reader.useSubscriptionManagerUserSettingsQuery();
-		return <SubscriptionManager.UserSettings value={ settings } />;
+		const query = Reader.useSubscriptionManagerUserSettingsQuery();
+		const { data: settings, isIdle, isLoading } = query;
```

Comment:
Just curiosity: what didn't you like about the initial approach? `const { data: settings, isIdle, isLoading } = Reader.useSubscriptionManagerUserSettingsQuery();`